config/packages/twig_extensions.yaml

```diff
@@ -4,8 +4,8 @@ services:
         autowire: true
         autoconfigure: true
 
-    # Uncomment any lines below to activate that Twig extension
```

Comment:
This shouldn't be changed.

Reply:
I'll be happy to merge this PR as soon as those lines are ignored by the tool.

Reply:
Hello,
I set line indent by his depths but comment line may not have any style. So, if comment line has nearest next non-comment line I set indent by that line but if comment line doesn't have non-comment line I set him zero indents because he does not belong to anyone.
I agree it's not perfect but I think it's the best solution. 
How exactly do you imagine logic to skip exactly these lines? 🤔 If you want all comment lines at the end set by nearest non-comment line then it can be not good for another repository where they write at the end some note, for example.

Sorry, if it looks aggressive but it's not 😄 I am only unhappy because I don't know how I can help you with this problem.

What do you think about edit that line to more indents and when you remove `#` they will have right indents? 🤔 
```yaml
services:
    _defaults:
        public: false
        autowire: true
        autoconfigure: true

#     Uncomment any lines below to activate that Twig extension
#    Twig\Extensions\ArrayExtension: ~
#    Twig\Extensions\DateExtension: ~
#    Twig\Extensions\IntlExtension: ~
#    Twig\Extensions\TextExtension: ~
```
instead of
```yaml
services:
    _defaults:
        public: false
        autowire: true
        autoconfigure: true

    # Uncomment any lines below to activate that Twig extension
    #Twig\Extensions\ArrayExtension: ~
    #Twig\Extensions\DateExtension: ~
    #Twig\Extensions\IntlExtension: ~
    #Twig\Extensions\TextExtension: ~
```
What do you think?

Reply:
Makes sense! Anyway, after giving it a bit more time, we don't really need this file at all in `Sylius/Sylius` repo :)